builtin/settingtypes.txt

```diff
@@ -1919,7 +1919,7 @@ lighting_boost_spread (Light curve boost spread) float 0.2 0.0 0.4
 #    Metrics can be fetched on http://127.0.0.1:30000/metrics
 prometheus_listener_address (Prometheus listener address) string 127.0.0.1:30000
 
-#    Maximum size of the out chat queue.
+#    Maximum size of the out-chat queue.
```

Comment:
Maybe "outgoing chat queue"? (If you change this, please also apply it to the short description below.)

Reply:
I like _outgoing_. Its antonym is _incoming_ which is also used in the setting: "Remove color codes from incoming chat messages".